finch/default.cfg

```diff
@@ -23,4 +23,5 @@ subset_threads = 1
 [logging]
 level = INFO
 file = finch.log
-format = %(asctime)s] [%(levelname)s] line=%(lineno)s module=%(module)s %(message)s
+database = sqlite:///pywps-logs.sqlite
```

Comment:
FYI, Finch WPS config in prod: https://github.com/bird-house/birdhouse-deploy/blob/master/birdhouse/config/finch/wps.cfg.template.  Maybe (not sure on this one) this `default.cfg` is used when a config is not specified in the override so be careful when you change config here.

In this case the `database` change is probably fine since we override with Postgres in prod.

Reply:
This change is not necessary anyway. 
Finch's wsgi app is created with this config file at the top and other passed config files are appended to the list.
I understand the later files have higher priority than the first one.